### DISCUSSION TITLE
Welcome to Habit-Tracker Discussions!

### DISCUSSION THREAD

**Discussion by Rd4dev** [Rd4dev/Habit-Tracker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sandy99405** [comment]
Hi @Rd4dev  My name is Sandilya.

I'm curious to know if you are still interested and invested in this project. Since I don't have any idea of my own I will be helping you with this.

To be honest I've discovered your profile through the **Google Summer Of Code (International Catrobat Association)** and thought of approaching you for a little guidance. 

But then I was very much impressed with your consistancy towards github contribution since the start of 2023.

Just like you, I have started my open source journey mid of 2023 when I had no idea about Open source. I just randomly searched and found some repos to work. 

I'm still figuring out everything by my own and want to learn more and more about android development, git and Open Source Contribution.

I've been wanting to use an app for maintaining my habits and during that time I found your **Habit-Tracker project**. I was very excited to contribute to it.

From now you will be seeing me contributing alot to this Project.

**Reply by Rd4dev** [reply]
Hi Sandilya,

Thank you for reaching out, and I appreciate your interest in contributing to the Habit-Tracker project! It's fantastic to connect with fellow open source enthusiasts. I'm treating this as a pet or passion project and would love to have open sourcers working on it to transform it into a fully usable project.

I will be more than happy to share insights on GSoC and answer any questions you may have about it.

Regarding the Bazel-related work on Oppia-Android, I've been working on it for the past two days and have a few questions. I believe it would be more convenient to connect on **[LinkedIn](https://www.linkedin.com/in/rd4dev/)** [please add a note so I can know its you] for a more in-depth discussion. Alternatively, if you could share your email, we can sort things out more effectively. I've been sharing GSoC content there as well; hopefully, that can also be helpful. 

Regards,
RD

**Reply by sandy99405** [reply]
Hey @Rd4dev  here is my linkedin profile [LinkedIn](https://www.linkedin.com/in/bukkapatnam-sandilya-4374aa204/). I've invited you
with a note and my email is sandysandilya99@gmail.com .......if you want me to connect through mail.
